dxa-framework/dxa-common-api/src/main/java/com/sdl/webapp/common/api/localization/Localization.java

```diff
@@ -33,6 +33,14 @@
     boolean isStaticContent(String url);
 
     /**
+     * Determines if the specified URL refers to static content within the webapp
+     *
+     * @return {@code true} if the specified URL refers to static content within the webapp, {@code false} otherwise.
+     * @param url a {@link java.lang.String} object.
+     */
+    boolean isNonPublishedAsset(String url);
```

Comment:
need default implementation that always returns false to not to break compatibility

Reply:
How would compatibility be broken exactly? As far as I can see there would only be a problem in the situation where the HTML files were published from the CMS _and_ you had put a _version.json file on the web app filesystem or dxa.assets.version property in dxa.properties - which would be a non-sensical situation, but still one which you would resolve by simply removing the file/property

Reply:
@willprice76 all these comments in all the PRs from are now for internal usage, so don't worry and ignore all of them :) We cannot merge all your PRs to 1.7 release on github, and we also have internal conventions on how to merge PRs from GitHub (happens through stash quality gate).

Regarding the comment itself: we have never defined what is really public API for DXA, so it doesn't hurt to add a default implementation to also support the case if anyone extended localization implementation or implemented their own localization.